zipkin/src/test/java/zipkin/storage/DependenciesTest.java

```diff
@@ -60,7 +61,7 @@ SpanStore store() {
 
   /** Clears store between tests. */
   @Before
-  public abstract void clear();
+  public abstract void clear() throws IOException;
```

Comment:
changing the base here is fine as you can override a method reducing its catch (especially as we're talking about test code)